config/software/gdbm.rb

```diff
@@ -17,14 +17,20 @@
 name "gdbm"
 default_version "1.9.1"
 
-source url: "http://ftp.gnu.org/gnu/gdbm/gdbm-1.9.1.tar.gz",
-       md5: "59f6e4c4193cb875964ffbe8aa384b58"
+version("1.9.1") { source md5: "59f6e4c4193cb875964ffbe8aa384b58" }
+version("1.11")  { source md5: "72c832680cf0999caedbe5b265c8c1bd" }
 
-relative_path "gdbm-1.9.1"
+source url: "http://ftp.gnu.org/gnu/gdbm/gdbm-#{version}.tar.gz"
+
+relative_path "gdbm-#{version}"
 
 build do
   env = with_standard_compiler_flags(with_embedded_path)
 
+  if version == "1.9.1" && ohai["kernel"]["machine"] == "ppc64le"
```

Comment:
I wonder if we should add some `ppc64le?` arch helper to chef-sugar? that would make these sorts of checks easier. We did something for the Solaris archs recently: https://github.com/sethvargo/chef-sugar/blob/master/lib/chef/sugar/architecture.rb#L44-L62